script.js

```diff
@@ -18,6 +18,28 @@ document.addEventListener("DOMContentLoaded", () => {
         
         servicio.addEventListener("mouseleave", () => {
             servicio.style.transform = "scale(1)";
+
+// Validación del formulario de contacto
+function validarFormulario(e) {
+    e.preventDefault();
+    
+    var nombre = document.getElementById('nombre').value;
+    var email = document.getElementById('email').value;
+    var mensaje = document.getElementById('mensaje').value;
+    
+    // Procesar formulario
+    console.log('Nombre:', nombre.toUpperCase());
```

Comment:
suggestion: Agrega trim() antes de toUpperCase() para eliminar espacios innecesarios al inicio y final del nombre.

​```suggestion
    console.log('Nombre:', nombre.trim().toUpperCase());
​```

Esto evita problemas con espacios que el usuario pueda ingresar accidentalmente y mejora la calidad de los datos procesados.

Reply:
Corregido en el commit más reciente.

Se ha agregado `.trim()` antes de `.toUpperCase()` en todos los console.log del formulario para eliminar espacios innecesarios. Esto mejora la calidad de los datos procesados.

Gracias por la sugerencia.